src/components/AccordionButton/AccordionButton.tsx

```diff
@@ -0,0 +1,81 @@
+import { Disclosure } from '@headlessui/react';
+import clsx from 'clsx';
+import React, { useContext } from 'react';
+import styles from './AccordionButton.module.css';
+import { ENTER_KEYCODE, SPACEBAR_KEYCODE } from '../../util/keycodes';
+import { AccordionContext } from '../Accordion';
+import Button from '../Button';
+import Icon from '../Icon';
+
+export type Props = {
+  /**
+   * Child node(s) that can be nested inside component.
+   */
+  children: React.ReactNode;
+  /**
+   * Additional classnames passed in for styling
+   */
+  className?: string;
+  /**
+   * Callback called when accordion is closed.
+   */
+  onClose?: () => void;
+};
+
+/**
+ * BETA: This component is still a work in progress and is subject to change.
+ *
+ * The interactive button subcomponent that houses the heading text and open indicator icon for the EDS Accordion component.
```

Comment:
Nit: "interactive button" seems a little redundant.

Reply:
True, will change to just `button`